trino/client.py

```diff
@@ -633,6 +633,9 @@ def process(self, http_response) -> TrinoStatus:
 
         self._next_uri = response.get("nextUri")
 
+        data = response.get("data", [])
```

Comment:
Probably would be more readable/less surprising if you remove the 2nd empty list arg to response.get.

I understand what happens is that the "data" key exists with null value so instead of returning empty list it returns None.

Reply:
Or maybe:

```
data = if response.get("data") else []
```